topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -58,6 +58,32 @@ We offer two ways to download the files. Firstly, you can download the files dir
 {% include _includes/cyoa-choices.html option1="EGA-Archive" option2="Zenodo" default="EGA-Archive"
        text="Here you can choose to either follow the data preperation for the data from the EGA-archive or Zenodo." %}
```

Comment:
Does anyone know how I can make sure to only show one of the pathways when using the code above?

Reply:
this should be the behaviour, only showing one. I will check it.